src/app/components/graph/graph-launcher.html

```diff
@@ -115,6 +115,12 @@ <h6>
                                         <ul
                                             class="dropdown-menu"
                                             ng-show="isVisible('resource_types')">
+                                            <li ng-click="onSelectAll('resource_types', allSelected ? false : true, $event)">
```

Comment:
Just curious - why do `allSelected ? false : true` instead of `!allSelected`?

Reply:
No particular reason I didn't think of that, but that is a bit cleaner I'll change it.